preview/MsixCore/msixmgr/Package.hpp

```diff
@@ -31,12 +31,16 @@ namespace MsixCoreLib
 
         /// Note: ExecutionInfo is populated by ProcessPSFIsNecessary() and may not be filled in on initialization.
         ExecutionInfo m_executionInfo;
+
+        /// Struct populated by ProcessPSFIsNecessary() and containing scriptrun information 
+        ScriptSettings m_scriptSettings;

```

Comment:
>m_scriptSettings [](start = 23, length = 16)

 = {}

Reply:
Initialized values to NULL, true, false, true and removed else case from package.cpp for the Boolean values.

Reply:
can these be initialized in IPackage.hpp instead? that way the name of the variable is present and better for readability

---
In reply to: [340916640](https://github.com/microsoft/msix-packaging/pull/249#discussion_r340916640) [](ancestors = 340916640)

Reply:
Initialized in IPackage.hpp